share/lua-build/luajit-2.0-dev

```diff
@@ -1 +1 @@
-install_git "luajit-2.0-dev" "https://github.com/luvit/luajit-2.0.git" master luajit
+install_git "luajit-2.0.5" "https://github.com/LuaJIT/LuaJIT.git" v2.0 luajit
```

Comment:
This should be a new file for 2.0.5. If -dev is gone this file can be deleted.

Reply:
Looks like 2.0.5 is already here

Reply:
`package_name` (this `luajit-2.0.5`) must be match with an executable binary built by package, otherwise a broken symlink `lua` is created at build phase.

```
% luaenv install luajit-2.0-dev
% ls -l $LUAENV_ROOT/versions/luajit-2.0-dev/bin
total 460
lrwxrwxrwx 1 who who     14 Apr 10 03:14 lua -> luajit-2.0-dev
lrwxrwxrwx 1 who who     12 Apr 10 03:14 luajit -> luajit-2.0.3
-rwxr-xr-x 1 who who 469144 Apr 10 03:14 luajit-2.0.3
```

(Note: Current luajit-2.0-dev source https://github.com/luvit/luajit-2.0.git is based on old luajit-2.0.3)

Reply:
Seems 2.0 dev maybe shouldn’t exist anymore? What do you think?

also if the binaries are 2.0.3 suffixed, should the package name be 2.0.3 the instead of 2.0.5?

Reply:
> Seems 2.0 dev maybe shouldn’t exist anymore? What do you think?

I agree.

> also if the binaries are 2.0.3 suffixed, should the package name be 2.0.3 the instead of 2.0.5?

Current source https://github.com/luvit/luajit-2.0.git is older than https://github.com/LuaJIT/LuaJIT.git v2.0 branch.
https://github.com/luvit/luajit-2.0.git is 2.0.3, https://github.com/LuaJIT/LuaJIT.git is 2.0.5.

Reply:
Can I include the commit removing  luajit-2.0-dev into this PR?

Reply:
Go for it